content/docs/how-we-work/git/git-workflow.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Git Workflow
+weight: 4
+description: Git branching and merging are essential concepts for managing any software. By using branches, we can isolate different features or bug fixes, and merge them back into the main branch as soon as they are ready.
```

Comment:
@Yemaneberhan-Lemma, the description is inaccurate

Reply:
Fixed.